### ISSUE TITLE
cleaned training sets

### ISSUE THREAD

**Issue by slala2121** [cgnorthcutt/label-errors#5]
Hi,

Are the cleaned training sets (from the Confident Learning paper in JAIR) and models that were trained on these datasets (cleaned training sets, cleaned validation sets) also available? Thanks.

**Comment by cgnorthcutt** [comment]
Hi, thanks for using cleanlab. Not everything could be uploaded due to size issues, but you can use this repo to recreate the cifar-10 models and experiments:
https://github.com/cgnorthcutt/confidentlearning-reproduce

For example, you can use the pyx file for zero noise to get the label errors in cifar 10 training data.

**Comment by slala2121** [comment]
Great, thanks I'll give that a try.
